packages/taiko-client/driver/chain_syncer/event/blocks_inserter/ontake.go

```diff
@@ -191,3 +191,8 @@ func (i *BlocksInserterOntake) InsertBlocks(
 
 	return nil
 }
+
+// not used
```

Comment:
So shall we just remove this method?

Reply:
yeah its a hangover from previous code, removed